src/app/components/MainMenu.js

```diff
@@ -0,0 +1,77 @@
+import createVisualImage from './VisualImage';
+import createLogo from './Logo';
+import createButtonRed from './ButtonRed';
+import createWhiteButtonWithIcon from './ButtonWhiteWithIcon';
+import createGameModeName from './GameModeName';
+import ModeMenu, { menuOption } from './ModeMenu';
+import createModeRules from './ModeRules';
+
+// Creating page content
+function PageContent(optionMode = { selectOption: 'People' }) {
+  const section = document.querySelector('.section');
```

Comment:
Is it possible to grab this node without implicit DOM usage? It's not a mandatory change, consider it as a tip